src/hotspot/share/gc/g1/g1ConcurrentRefine.cpp

```diff
@@ -243,15 +252,17 @@ static size_t calc_min_yellow_zone_size() {
 
 static size_t calc_init_green_zone() {
   size_t green = G1ConcRefinementGreenZone;
+  char* name = (char*) "G1ConcRefinementGreenZone";
```

Comment:
Change the type of name to `const char*` and eliminate the cast here and on line 258.

Reply:
Amazing!
It gets compiled without the casts by just adding 'const'.

Updated. Thanks.